crates/core/src/db/relational_db.rs

```diff
@@ -494,7 +494,6 @@ impl RelationalDB {
         tx: &'a MutTx,
         table_id: TableId,
     ) -> Result<Iter<'a>, DBError> {
-        let _guard = DB_METRICS.rdb_iter_time.with_label_values(&table_id.0).start_timer();
```

Comment:
This metric was was used, but it did not measure actual iteration time, just the time to open an iterator.